Monika After Story/game/script-topics.rpy

```diff
@@ -12905,3 +12905,205 @@ label monika_enjoyingspring:
     else:
         m 2rkc "...but I guess there's no real way to avoid it, is there?"
     return
+
+# start of bookmark and derandom related events
+default persistent._mas_player_bookmarked = {}
+# dict to store bookmarked events
+default persistent._mas_player_derandomed = {}
+# dict to store player derandomed events
+default persistent.flagged_monikatopic = None
+# var set when we flag a topic for derandom
+
+init python:
+    def derandom_topic():
```

Comment:
would be nicer if you made this (and the bookmark_topic) functions take in an ev_label so we can directly call these if we elsewhere rather than having to set the current_monikatopic and then calling this. The hotkey versions can just pass in the curent_monikatopic instead.

Also use the `mas_` prefix since these are in the global store.